build/patches/Fix-segmentation-platform-build-error.patch

```diff
@@ -0,0 +1,22 @@
+From: csagan5 <32685696+csagan5@users.noreply.github.com>
+Date: Sun, 16 Oct 2022 08:42:55 +0200
+Subject: Fix segmentation platform build error
+
+License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
+---
+ components/segmentation_platform/embedder/BUILD.gn | 1 -
```

Comment:
how come a new one and not inside the `Remove-segmentation-platform.patch`?
I put it there.

Reply:
It is needed to build Chromium.

Reply:
As always, I explain myself badly.
I am saying that I think you can put it in Remove-segmentation-platform.patch instead of making another one.

Reply:
`Remove-segmentation-platform.patch` is not used to build Chromium so this change cannot be there because it is needed to build Chromium.